components/infobox/commons/infobox_person.lua

```diff
@@ -24,187 +23,211 @@ local function GetBirthAndDeath()
-			)
-			:cell('Region', self:_createRegion(args.region))
-			:fcell(Cell	:new('Team')
-						:content(
-							self:_createTeam(args.team, args.teamlink),
-							self:_createTeam(args.team2, args.teamlink2)
-						)
-						:make()
-			)
-			:fcell(Cell	:new('Clan')
-						:content(
-							self:_createTeam(args.clan, args.clanlink),
-							self:_createTeam(args.clan2, args.clanlink2)
-						)
-						:make()
-			)
-	infobox:cell('Alternate IDs', args.ids or args.alternateids)
-	infobox:cell('Nicknames', args.nicknames)
-	infobox:cell('Total Earnings', earnings)
-	self:addCustomCells(infobox, args)
 
-	local links = Links.transform(args)
-	local achievements = self:getAchievements(infobox, args)
-	local history = self:getHistory(infobox, args)
-
-	infobox:header('Links', not Table.isEmpty(links))
-	infobox:links(links, _LINK_VARIANT)
-	infobox:header('Achievements', achievements)
-	infobox:centeredCell(achievements)
-	infobox:header('History', history)
-	infobox:centeredCell(history)
-	infobox:centeredCell(args.footnotes)
-	self:addCustomContent(infobox, args)
-	infobox:bottom(self:createBottomContent(infobox, args))
-
-	if _SHOULD_STORE_DATA then
-		args.birthDisplay = birthDisplay
-		self:getCategories(infobox, args, role.category, status.store)
-
-		links = Links.makeFullLinksForTableItems(links, _LINK_VARIANT)
-		local lpdbData = {
-			id = args.id or mw.title.getCurrentTitle().prefixedText,
-			alternateid = args.ids,
-			name = args.romanized_name or args.name,
-			romanizedname = args.romanized_name or args.name,
-			localizedname = args.name,
-			nationality = args.country or args.nationality,
-			nationality2 = args.country2 or args.nationality2,
-			nationality3 = args.country3 or args.nationality3,
-			birthdate = birthday,
-			deathdate = deathday,
-			image = args.image,
-			region = args.region,
-			team = args.teamlink or args.team,
-			status = status.store,
-			type = role.store,
-			earnings = earnings,
-			links = links,
-			extradata = {},
-		}
-		lpdbData = self:adjustLPDB(lpdbData, args, role, status)
-		lpdbData.extradata = mw.ext.LiquipediaDB.lpdb_create_json(lpdbData.extradata)
-		lpdbData.links = mw.ext.LiquipediaDB.lpdb_create_json(lpdbData.links)
-		local storageType = self:getStorageType(args, role, status)
-
-		mw.ext.LiquipediaDB.lpdb_player(storageType .. self.name, lpdbData)
+	local widgets = {
+		Header{name = self:nameDisplay(args), image = args.image},
+		Center{content = {args.caption}},
+		Title{name = (args.informationType or 'Player') .. ' Information'},
+		Cell{name = 'Name', content = {args.name}},
+		Cell{name = 'Romanized Name', content = {args.romanized_name}},
+		Cell{name = 'Birth', content = {birthDisplay}},
+		Cell{name = 'Died', content = {deathDisplay}},
+		Customizable{id = 'status', children = {
+			Cell{name = 'Status', content = { args.status }
+				}
+			}
+		},
+		Customizable{id = 'roleorfaction', children = {
+			Cell{name = 'Role', content = { args.role }
+				}
+			}
+		},
```

Comment:
This doesn't exist on a whole bunch of wikis, so not convinced this should be here. Role is common, but I don't see how it combines with "faction"?

Reply:
because the games that have the role do not have faction and the games that have faction do not have the ingame roles
i mean we for sure can rename the id to `role`, but i still would like to override it with the faction stuff on sc/sc2

Reply:
Role is used for things like coach though, so it has no relevance to in-game preferences at all. Shouldn't these just be separate?

Reply:
on most wikis (at least of those that i checked) role is used for ingame role too:

it is quite a mess imo

examples (from the template docs):
cs:
![Screenshot 2021-09-09 16 56 17](https://user-images.githubusercontent.com/75081997/132710605-5027c1b1-2fd5-48a3-9ff4-03065ba90675.png)
dota2:
![Screenshot 2021-09-09 16 56 43](https://user-images.githubusercontent.com/75081997/132710614-1f9cfcbd-680c-4dc6-9337-4f342ea34d01.png)
lol:
![Screenshot 2021-09-09 16 57 13](https://user-images.githubusercontent.com/75081997/132710616-6cbbbfe3-c1a5-44b0-80af-3f2a9c39d7ab.png)

Reply:
I see. I think we leave these out then and consider them all custom cells